improver/standardise.py

```diff
@@ -267,9 +234,54 @@ def _remove_scalar_coords(cube, coords_to_remove):
             except CoordinateNotFoundError:
                 continue
 
+    @staticmethod
+    def _standardise_dtypes_and_units(cube):
```

Comment:
I think it might be useful if this has its own unit tests

Reply:
I'm not sure you have a unit test for converting a cube's data type. I think the test that check this does the right thing for coordinates (test_standardise_time_coords) doesn't actually check the points are correct after the conversion.

Reply:
It is there. I've made it clearer and added a test for int64 deescalation to int32 too.